src/schema.py

```diff
@@ -101,7 +101,7 @@ class BackendStatusDatasetsSchema(BaseModel):
 
 class ServerSchema(BaseModel):
     version: str
-    auth_enabled: Optional[str]
+    auth_enabled: Optional[bool]
```

Comment:
I suspect this will cause issues on the frontend (but you can check). Unfortunately, you should probably cast to str when creating ServerSchema.